src/argilla/server/search_engine/commons.py

```diff
@@ -114,15 +124,58 @@ class Config:
         getter_dict = SearchDocumentGetter
 
 
-def index_name_for_dataset(dataset: Dataset):
+def es_index_name_for_dataset(dataset: Dataset):
     return f"rg.{dataset.id}"
 
 
-def field_name_for_vector_settings(vector_settings: VectorSettings) -> str:
+def es_terms_query(field_name: str, values: List[str]) -> dict:
+    return {"terms": {field_name: values}}
+
+
+def es_range_query(field_name: str, gte: Optional[int] = None, lte: Optional[int] = None) -> dict:
```

Comment:
In `RangeFilter` we are using `float` instead of `int` for `ge` and `le`. Maybe you need to use `float` here as well?

```suggestion
def es_range_query(field_name: str, gte: Optional[float] = None, lte: Optional[float] = None) -> dict:
```